stl/inc/variant

```diff
@@ -39,6 +39,12 @@ struct _All_same : true_type {};
 template <class _First, class... _Rest>
 struct _All_same<_First, _Rest...> : bool_constant<conjunction_v<is_same<_First, _Rest>...>> {}; // variadic is_same
 
+template <class...>
+struct _All_convertible : true_type {};

```

Comment:
I don't think `_All_convertible<>` with no target type makes sense as does `_All_same<>`. I suggest dropping the above and making the below the primary template.